### PR TITLE
Adds Ithildin material for future use of plates in the recipes of AB

### PR DESCRIPTION
**Pull request by EnderProyects**
- Add new materials for future use, and for changing ingots in recipes with the pates, rods or any other item. 
- Set durability and speed of this item a little higher from gaia, since is an upgraded version 
- Change the name from Mithrill to Botanic-Mithril or a similar name I would like Mithrilium or Ithildin @Alastors do you like this name?

### CODE REVIEW

File: src/main/java/gregtech/api/enums/MaterialsBotania.java [hunk 1]
```diff
@@ -44,6 +44,26 @@ public class MaterialsBotania {
                 new TC_AspectStack(TC_Aspects.TERRA, 1),
                 new TC_AspectStack(TC_Aspects.PRAECANTATIO, 1)))
         .constructMaterial();
+    public static Materials MagicMithrill = new MaterialBuilder(210, TextureSet.SET_METALLIC, "Magic Mithrill")
+        .setName("MagicMithrill")
```
Comment (Alastors) [inline src/main/java/gregtech/api/enums/MaterialsBotania.java:48]: Spell it right, or don't use Mithril at all

Comment (Alastors) [inline src/main/java/gregtech/api/enums/MaterialsBotania.java:48]: Spell it right or don't use mithril at all

File: src/main/java/gregtech/api/enums/MaterialsBotania.java [hunk 2]
```diff
@@ -44,6 +44,26 @@ public class MaterialsBotania {
...
+        .setRGBA(15, 232, 198, 1)
+        .addDustItems()
+        .addMetalItems()
+        .addToolHeadItems()
+        .addGearItems()
+        .setToolSpeed(40.0F)
+        .setDurability(1000000)
+        .setToolQuality(12)
+        .setMeltingPoint(5400)
```
Comment (Alastors) [inline src/main/java/gregtech/api/enums/MaterialsBotania.java:57]: What exact tier is this supposed to be, you give no balance reasoning to this

File: src/main/java/gregtech/api/enums/MaterialsBotania.java [hunk 3]
```diff
@@ -44,6 +44,26 @@ public class MaterialsBotania {
                 new TC_AspectStack(TC_Aspects.TERRA, 1),
                 new TC_AspectStack(TC_Aspects.PRAECANTATIO, 1)))
         .constructMaterial();
+    public static Materials MagicMithrill = new MaterialBuilder(210, TextureSet.SET_METALLIC, "Magic Mithrill")
```
Comment (Alastors) [inline src/main/java/gregtech/api/enums/MaterialsBotania.java:47]: Fix the name

**Review by Alastors — CHANGES_REQUESTED** [review]
This is unnecessary as it is, you are adding a new misspelt name to a set of materials that already has 3 mithrils, either drop the extra L and allow the "magic" to be the differentiator, (which is kind of cringe, I'd call it Botanic Mithril or Elven Mithril) or don't use the name mithril at all.

**Comment by Alastors** [comment]
Likewise, I can't stress this further, **_post notes regarding your pr to justify the reasoning behind the pr, don't just submit blank prs_**

File: src/main/java/gregtech/api/enums/MaterialsBotania.java [hunk 4]
```diff
@@ -44,6 +44,26 @@ public class MaterialsBotania {
...
+        .setName("MagicMithrill")
+        .setRGBA(15, 232, 198, 1)
+        .addDustItems()
+        .addMetalItems()
+        .addToolHeadItems()
+        .addGearItems()
+        .setToolSpeed(40.0F)
```
[the remark this answers is not in the document]
Comment (EnderProyects) [reply src/main/java/gregtech/api/enums/MaterialsBotania.java:54]: Suggest a tier, it should be a little up from gaia

[the remark this answers is not in the document]
Comment (Alastors) [reply src/main/java/gregtech/api/enums/MaterialsBotania.java:54]: This is your pr, not mine, propose a tier or don't post it in the first place

**Comment by BritishCynic** [comment]
Closing as contributor has no interest in engaging in good faith, nor have they done appropriate research on where this material should be placed in the tiering system. Until the above points of concern have been addressed and suitable suggestions from the contributor have been obtained on tiering, this PR should remain closed.

**Comment by Dream-Master** [comment]
I reopen this now. If a pr is not good yet it can be changed/adjust or closed by Original author .

**Review by Alastors — CHANGES_REQUESTED** [review]
As has been brought forth by most of the dev team now, I think instead of pushing this forward as a new material, we should close this pr and instead repurpose the already existing yet useless Mithril already found in the GT material list. 

It's not really a justification to add a new material by stating other mods already have unused metals, we're trying to move away from our history of incessant bloat, not continue this into the far flung future.

Wouldn't you agree @Dream-Master?

**Comment by Dream-Master** [comment]
@Alastors chat with @EnderProyects or talk with him. True we have a lot unsed materials in the game already. I am sure you and Ender found a solution.

**Comment by Alastors** [comment]
> @Alastors chat with @EnderProyects or talk with him. True we have a lot unused materials in the game already. I am sure you and Ender found a solution.

The solution pushed forth by @boubou19 and @mitchej123 amongst others was to just poach the existing mithril and repurpose it for this very thing, I don't think it needs more of a solution than that personally.

**Comment by Alastors** [comment]
This has already been hashed out endlessly over two days, I don't see how this conversation can go any further with how much of the dev team is against adding this material.

**Comment by boubou19** [comment]
Well looks like a conclusion has been reached: @EnderProyects just use Mithril.

**Comment by EnderProyects** [comment]
> > @Alastors chat with @EnderProyects or talk with him. True we have a lot unused materials in the game already. I am sure you and Ender found a solution.
> 
> The solution pushed forth by @boubou19 and @mitchej123 amongst others was to just poach the existing mithril and repurpose it for this very thing, I don't think it needs more of a solution than that personally.

I´m okay with this, but then the Mithrill ore should be removed and the recipes where is used it should be tweaked

**Comment by boubou19** [comment]
That will not happen before AB is finished

**Pull request closed without merging**